fighthealthinsurance/templates/admin_status.html

```
@@ -0,0 +1,199 @@
+{% load static %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)